focus_areas/code_development.md

```diff
@@ -24,27 +24,64 @@ to time periods.
 
 Goal **Activity**:
 
-* Question **Changes**: How many changes are happening to the code base, during a certain time period? 
+* Question **Changes**: How many changes are happening to the source code, during a certain time period? 
 
-  * Metric **Code_Changes_No**(Period): Number of changes to the code base (see [Code_Changes](../metrics/Code_Changes.md))
-  * Metric **Code_Lines_Changed_No**(Period): Number of lines changed in the source code
-  * Metric **Code_Changes_Lines_No**(Period): Aggregated number of lines touched in all changes.
+  * Metric **Code_Changes**(Period): Number of changes to the source code
+  (see [Code_Changes](../metrics/Code_Changes.md))
+  * Metric **Code_Changes_Lines**(Period): Aggregated number of lines touched in all changes.
 
-* ...
+* Question **Proposals**: How many proposals for changes to the source code
+are happening during a certain time period?
 
-Goal **Efficiency**:
+  * Metric **Proposals**(Period): Number of new proposals for changes
+  to the source code.
+  * Metric **Proposals_Accepted**(Period): Number of proposals for changes
+  to the source code that were accepted.
+  * Metric **Proposals_Declined**(Period): Number of proposals for changes
+  to the source code that were declined.
+
+* Question **Issues**: How many issues related to the source code
+are happening during a certain time period?
```

Comment:
Since GitHub and GitLab are dominant git hosts in the world, I think it would be a good idea to make a comment about the specific implementation of "proposals" for each platform, noted in my previous comment.